src/renderer/rendered_window.rs

```diff
@@ -146,7 +148,7 @@ impl RenderedWindow {
             return destination;
         }
 
-        let mut grid_size: GridSize<f32> = self.grid_size.cast();
+        let mut grid_size: GridSize<f32> = self.grid_size.try_cast().unwrap();
```

Comment:
is there no cast function that panics? this pattern seems odd

Reply:
To my it looks pretty normal, it uses this under the hood https://docs.rs/num/latest/num/trait.NumCast.html#. The built in `TryInto/TryFrom` are also similar, but they return a result instead.

There is a non-failing cast for converting just the units, https://docs.rs/glamour/latest/glamour/struct.Point2.html#method.cast, but it does not apply here, since we are converting from int to float.

There's also `into` which can convert between vector and point for example.

As far as I know there are no non-failing cast, similar to `as` unless you convert it to something else first.